aps/toolbox/example_scripts/test_resample_properties_from_ertbox1.py

```diff
@@ -2,14 +2,17 @@
 # to copy 3D parameters from ertbox grid to geogrid.
 
 
-from aps.toolbox import copy_rms_param_to_ertbox_grid
-from aps.utils.constants.simple import Debug
+# TODO: When fmu.tools is merged into komodo stable,
+# switch to fmu.tools and remove copy_rms_param_to_fmu_grid from aps.rms_jobs
 
-print(f'Run script: {copy_rms_param_to_ertbox_grid.__file__}  ')
-print(f'Copy 3D parameters from Ertbox grid to geo grid')
+from fmu.tools.rms import copy_rms_param
+
+
+print(f'Run script: {copy_rms_param.__file__}  ')
+print('Copy 3D parameters from Ertbox grid to geo grid')
 params = {
     'project': project,
-    'debug_level': Debug.ON,
+    'debug_level': 1,
```

Comment:
We should use the enum when possible to be clear what the intension is

Reply:
OK